webapp/packages/core-ui/src/Form/FormState.ts

```diff
@@ -211,6 +214,7 @@ export class FormState<TState> implements IFormState<TState> {
 
   async save(): Promise<boolean> {
     try {
+      this.isSaving = true;
       this.isDisabled = true;
```

Comment:
I beleive, we can replace this prop with `isSaving` then all over where it used

Reply:
fixed